saas/static/js/djaodjin-saas-vue.js

```diff
@@ -871,41 +871,97 @@ var filterableMixin = {
     },
 }
 
+var DESC_SORT_PRE = '-';
+
 var sortableMixin = {
     data: function(){
+        var defaultDir = djaodjinSettings.sortDirection || 'desc';
+        var dir = (defaultDir === 'desc') ? DESC_SORT_PRE : '';
+        var o = djaodjinSettings.sortByField || 'created_at';
         return {
             params: {
-                o: djaodjinSettings.sortByField || 'created_at',
-                ot: djaodjinSettings.sortDirection || 'desc',
+                o: dir + o,
             },
             mixinSortCb: 'get'
         }
     },
     methods: {
-        sortBy: function(fieldName) {
-            if(this.params.o === fieldName) {
-                if(this.params.ot === "asc") {
-                    this.params.ot = "desc";
-                } else {
-                    this.params.ot = "asc";
+        sortDir(field){
```

Comment:
Replace by   `sortDir: function(field){` otherwise vuejs code is not executed on phantomjs. I believe it might be JS6 related syntax or it is just that Chrome is a lot more lenient.